serverlessrepo/publish.py

```diff
@@ -3,9 +3,13 @@
 import boto3
 from botocore.exceptions import ClientError
 
+from .application_metadata import ApplicationMetadata
 from .parser import parse_template, get_app_metadata, parse_application_id
 
 SERVERLESSREPO = boto3.client('serverlessrepo')
```

Comment:
Question: It is probably in previous PR and I missed it. How do you configure which region the client will create resources in?